src/windows/volume.cc

```diff
@@ -18,6 +18,32 @@
 
 static const size_t kVolumePathShortLength = 4;
 
+HRESULT drivelist::volume::GetSystemVolume(wchar_t *out) {
+  const char * const systemDrive = getenv("SystemDrive");
```

Comment:
I wonder if there's a more reliable way of determining this, than checking a particular environment variable?
After following a bunch of deprecated-links from https://stackoverflow.com/questions/810273/how-can-i-get-the-system-drive-letter I eventually ended up at https://msdn.microsoft.com/en-us/library/bb762188.aspx and https://msdn.microsoft.com/en-us/library/dd378457(v=vs.85).aspx which I guess suggests we could check `FOLDERID_System` ?